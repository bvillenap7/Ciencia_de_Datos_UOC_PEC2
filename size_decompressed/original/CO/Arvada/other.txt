Two objects seen (daytime) over Arvada Colorado;2008-06-24 09:10:00;10 min;Two objects seen (daytime) over Arvada Colorado Looking west two object were seen. One large one small.  The larger object was square brown and mostly transparent with spots.   Although the larger object was square it had the characteristics of an amebae organism.  The second object was much smaller and very close buy the larger object.  The second object appeared to be of a very shinny silver material but extremely thin. The second object was floating horizontal.  The object was first spotted in the western sky by the first person.  Person # 1 called a coworker over and both saw the larger transparent object but the coworker could not see the thin second object.  The objects were stationary and the wind made the larger object wave like a flag because the object was flat.  There were NO aircraft around what-so-ever.  The thin silver object seemed to be pulling or holding the larger object.  A large very dark shadow moved over the building and when the witnesses looked back to wear the object they were both gone.  No clouds and a very windy day.  The event left a 'feeling' or impression with one of the witnesses that there was something wrong the event.  Something wasn't right.;39.8289982352941;-105.110888235294
bright orb seen flying over arvada, colorado;2011-01-24 06:12:00;10 minutes;bright orb seen flying over arvada, colorado This object came out of the north/northwest.  It was a circular ball of light that I thought at first might be an aircraft landing light.  It was as bright as an aircraft light,very bright.  However there was no flashing lights of any kind.  I did have binoculars, but could only make out a bright orb.  It moved directly over me as it was performing a wide arcing turn. It eventually was headed toward the east/northeast and I watched it until it disappeared.  The total event lasted around 10 minutes time.  It made no sound whatsoever and never varied its speed or altitude.  It never lost brightness only dimmed as it receded into the distance.  Its altitude didn't seem as high up as a commercial jet, but I couldn't be sure because of nothing to compare it to. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
Arrow shaped object w/dim lights.  ((anonymous report));2016-10-07 21:05:00;10 seconds;Arrow shaped object w/dim lights. I saw what appeared to be a large arrow shaped object flying N to S.   It had very dim, almost translucent amber colored lights.   There was no sound and it moved at a quick and steady pace. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
Scarey stuff over colorado.  ((anonymous report));2019-08-06 23:35:00;10 seconds of my view;scarey stuff over colorado as i walked to the trash i looked west and saw a half square half circle shape in the sky.  very large.  bright like the sun but sky was still dark.  i thought it was the moon at first but quickly realized the shape and speed.  it disappeared from view.  the shape never changed. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
See above.   ((Starlink satellites?));2020-03-05 19:50:00;5 minutes;See above Around 7:50-7:55 pm MT, me and my children ages 8 and 11, witnessed strange lights, (looked like dense stars) moving south west of Venus towards northeast of Venus up to the sky.  There were about 8 of them in a row.... and they were spaced almost evenly heading diagonally upwards in orbit of earth.;39.8289982352941;-105.110888235294
Hemishphere that looked like the moon at first, but appeared to be 4x larger and sideways on 64th ave in arvada. Retreated into the sky;2020-11-26 03:00:00;30 seconds;Hemishphere that looked like the moon at first, but appeared to be 4x larger and sideways on 64th ave in arvada. Retreated into the sky and disappeared.;39.8289982352941;-105.110888235294
