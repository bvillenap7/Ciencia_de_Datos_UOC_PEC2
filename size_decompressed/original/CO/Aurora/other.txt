Low, hovering object that appeared to be an airplane, but was too low and slow in the sky, then disappeared.;2004-03-17 18:00:00;2 minutes;Low, hovering object that appeared to be an airplane, but was too low and slow in the sky, then disappeared. My four daughters and I were traveling east on Alameda in our Suburban. In the distance we saw a strange airplane which appeared to be ascending from the ground where there is no airport, south to north. It turned and came straight toward us, traveling west on Alameda. It appeared to be an airplane, flying very slowly and low in the sky, but it stopped and hovered above and to the north of us. We saw other drivers watching, open mouthed at the same time. It seemed to just stop in the air, wasn't as high as it should have been for the size of the aircraft. It tilted on its left wing, then I drove past it as it was unusual and scared me, seemed it might crash. Looked in my rearview and the girls looked out the back window, but it was gone. Just disappeared. We couldn't see it anywhere behind or in front of us. It looked metallic, gray, kind of like a fighter airplane, but did not act or sound like one. I have no idea what it was.;39.6695444272445;-104.803991486068
Multiple (7-15) triangle shaped crafts with lights that would flash different colors;2015-07-16 02:30:00;3 hours;Multiple (7-15) triangle shaped crafts with lights that would flash different colors They started out as very shinny lights in the sky, me and a friend where on his balcony and noticed them.  When we took a closer look at them they seemed to look like they where growing and shrinking vertically, it was all different colors flashing mostly consisting of green, blue, white and red. They moved like they were flying in formation. One would move and then stop and it kept repeating itself until it eased its way across the sky.  Some of them would break formation suddenly and fly somewhere unrelated to its former path. They seemed to be right over Buckley Airforce Base.  Things got weird when two very silent jets took off from Buckley Airforce base and never came back from what we saw.;39.6695444272445;-104.803991486068
I'm not even sure what really happened.  ((NUFORC Note:  We are not certain this is intended as a serious report.  PD));2015-12-13 23:00:00;48:37 minutes;I'm not even sure what really happened We were just sitting and hanging out.  We had sneaked out that night and our parents didn't know about it.  We were just hanging out having a good time when suddenly we all couldn't move.  I could only move my eyes.  I looked around and everyone was completely still with their eyes moving from side to side.  Out of nowhere came a loud shrill sound from no where.  Our immobile mouths only made shrill cries from our throats.  It wouldn't stop. When it finally did and we could move again it was all still.  Then out of nowhere a large spot of light appeared in the sky.  It appeared almost like the sky was a rag soaking up the liquid-is state of the ship.  It shared no certain shape but it was so bright it burnt my eyes almost like staring at the sun, and that's all we could do was stare in ah  (sic.  awe).  Eventually a loud boom expelled from the light and then another and then another and we all scrambled for our cars.  The light followed us as we ran and even followed us till we were! suddenly all about 60 miles outside the city limits.  I was dropped off at home by my partner who i had spent the night with.  When i woke up this morning I found an extremely large bruise on the complete center of my chest.  My friends have also said they have also found large perfectly circular bruises on different parts of their body.  I'm extremely shaken up and needed to tell someone about these events. ((NUFORC Note:  We are not certain this is intended as a serious report.  Source of report elects to remain totally anonymous, and provides no contact information.  PD)) ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
Bright white shape seen close to a park.;2016-09-18 19:00:00;~1-2 minutes;Bright white shape seen close to a park. The object was bright white, almost as if it was glowing or shining. It was heading westward towards the setting sun. The best way I can describe it is a four pointed star with the top point chopped off.  I know this sounds weird, but I tried to take a picture and it would not show up on my camera.  I was using only an iPhone camera and it was very far away by that point, but it was still very strange. The object made no noise, however, there was an airplane nearby in the eastern hemisphere of the sky as I live very close to an airport.  The objects were not interacting, as far as I could see.;39.6695444272445;-104.803991486068
Single white crescent shaped light soundlessly moving 80 feet above me.;2017-08-14 21:00:00;5 seconds;Single white crescent shaped light soundlessly moving 80 feet above me I had gone out to the back yard with my dog and happened to look straight up.  I saw a crescent shaped light with a span of 5-6 feet, sharply defined against the dark cloudless sky approximately 80 feet above me.  It was moving from south to north in a straight line and made no sound.  Once I saw it and registered all of its detail, the light simply went out.   I continued to watch the spot where it had been but it did not reappear.  Once I was back inside, I wanted to check for any similar reports, but my internet was out.  Interestingly, the internet subscribers nearby were displayed and all of them had full range service.   My internet service came on 1/2 hour later.;39.6695444272445;-104.803991486068
The lights always seem to make all the neighborhood dogs bark.  ((NUFORC Note:  Possible sighting of a planet??  PD));2017-12-22 01:00:00;Several hours;The lights always seem to make all the neighborhood dogs bark. Also when the lights appear I get a massive headache and feel nauseous. I WAS WOKEN UP BY MY DOGS AND OPENED THE CURTAINS TO MY ROOM WIDE OPEN. IT WAS CLEAR SKIES AND THE MOON WAS BRIGHT,  I HAPPENED TO LOOK SOUTH EAST OF MY WINDOW AND TWO lighted OBJECTS MOVING UP AND DOWN AND SIDE WAYS.  I THOUGHT MY EYES WERE PLAYING TRICKS ON ME, SO I put my glasses on to take a better look and it was still moving around.  The left one was moving more then the right one.  I felt that I must be crazy, so I woke my boyfriend up and told him to look up towards the SE skies and he was looking at the the lights moving oddly as well.  He went back to sleep because he does not like to be woken up like that, but I kept looking at the lights until I fell asleep.   This has not been the first time I have seen lights move around like that.  About 2 years ago it happened similar like that but no one one believes me.  When this occurs I get a massive headache and feel nausea. ((NUFORC Note:  Possible sighting of a planet??  PD));39.6695444272445;-104.803991486068
flying objects changing colors, animals reacted, magnetic field.;2020-09-17 21:00:00;2.5 hours;flying objects changing colors, animals reacted, magnetic field. so i had been in my room and i was looking out my window and is saw about 7 different lights blinking at once different colors i would say at about 70 mph and they had split up and one had stayed and than it went up and vanished.;39.6695444272445;-104.803991486068
