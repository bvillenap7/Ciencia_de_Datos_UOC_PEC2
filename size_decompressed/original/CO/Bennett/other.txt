I was outside my house watching the star. It had been the first night for a while I could see them so clear. I was looking north over t;2008-08-01 22:45:00;1 minute;I was outside my house watching the star.  It had been the first night for a while I could see them so clear. I was looking north over the top of our house there were several planes towards the south where the regional airport is and DIA. I saw a bright light which I believed to be another plane coming from the north west towards southeast. I was watching it and realized there were no lights around it flashing like a plane and it was going faster then a plane but not as fast as a falling star and it was heading more up then down like a star would. I yelled at my wife to take a look but she didn’t hear me. The object(light) then started fading till it almost disappeared then became a red streak then disappeared. There were no clouds in the sky so it didn’t go behind or above a cloud. That was all I saw from that point on. The whole thing probably took a minute or two.;39.7569;-104.4266
