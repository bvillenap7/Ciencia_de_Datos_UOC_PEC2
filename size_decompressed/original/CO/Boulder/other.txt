This object looked like a bright star, zig-zagging, abruptly stopping & then zig-zagging @ a high rate of speed above Boulder, Colorado;1969-06-01 01:00:00;5mins;This object looked like a bright star, zig-zagging, abruptly stopping & then zig-zagging @ a high rate of speed above Boulder, Colorado. This event took place while camping above Boulder, CO. on Flagstaff Mountain.  At this position we had a very clear view of the sky & the town below. The object went from one end of the sky to the other & was witnessed by myself as well as 4 other friends. This took place approximately in  July of 1969. We were all just kids & no one would believe us. Since then I've been in the military & have traveled abroad & have witnessed another sighting in Terrell TX. The object was copper in color , it came out of the woods as I was driving down the road late @ night & I lost sight of it as it shot straight up. This happened in 1978 on or around the month of May right off highway 429. This was an unpopulated area @ the time. ((NUFORC Note:  Date is approximate.  Witness records 1970, but states that the incident occurred in 1969.  We have amended the date here.  PD));40.0217727272726;-105.258365102639
I witnessed a shape over head coming from the north;1998-09-13 22:10:00;10 seconds;I witnessed a shape over head coming from the north,which i thought was a cloud at first,then as it got closer ,i thought it was a hanglider.It being 10 at night,i realized i was seeing something very unusual.I was playing a song,on  guitar, in my backyard,when i saw the object.I stood up from my chair,and watched it pass directly over head,moving from north to south.I believe it was boomerang shaped,although it may hve been cylindrical.It had no lights,and a pewter or lead kind of sheen.It almost seemed to blend in with the sky. It also seemed to change shape ,as i was watching it move over head.I felt a most incredable rush,as i have been a believer in ufo's,though have never actually seen one,till now !! The craft disapeared behind a cloud,as i went imediatly inside to take notes ,and sketch out what i had seen;40.0217727272726;-105.258365102639
Dark metalic object along Colorado Frontrange;1999-08-15 13:00:00;10 minutes;Dark metalic object along Colorado Frontrange While attending school at U of C, I painted houses as a summer job, in and around the Boulder area.  We (a fellow employee and I)were painting a house about 10 min north of Boulder, wich was in an older golf course type communitee along the foothills, I would say we were about 2 miles east of the frontrange.  While painting I noticed a black object appear from over the peaks slightly Northwest of our location, it seemed to have risen from behind the hills, but given our location was far below mountain top, it may have been travelling west at a steady altitude before it was visible to us.  After clearing the mountain top, the object lowered altitude slightly, to,I would say between 300-500ft. and continued to move east.  As it continued our way, but slightly northwest of us, it would travel slowly, periodicaly jolting forward, but never changing altitude.  We watched as it traveled from the mountain range, at first from a distance it did look as though it could have been a very large balloon, and was a dark metallic color (like Black Chrome... but not as shiney) but was not round or teardrop , as it got closer it looked a bit like a pyramidic bubble, rotating randomly on no certain axis, and possibly the size of a school bus (this would be at a distance less than a quarter mile north of us, and at the same altitude stated, I know we had to look upward)  as it passed it seamed to pick up speed and headed more northeast, it must have begun moving much faster, because it was out of sight quicker than the time it took to reach our vacinity from the mountains.;40.0217727272726;-105.258365102639
W shaped UFO sighting in Colorado.;2007-12-03 08:00:00;Driving;W shaped UFO sighting in Colorado. I was driving around 8 o'clock at night. And I looked up in the sky, and I saw somthing flying rather low. At first I thought it was a plane, because there are public runways for planes to take off and land near where I was. But I took a closer look, and I saw strange lights. there were about 7-10 lights. some were flickering dimmly. and some were remaned on. There were multied colors of the lights. they all changed. some went to red, to white, to blue. and then brown, orange. Pink, Yellow. (Very Wired.) The lights were forming the UFO's shape as a W. It was dark out, so I wasnt compleatelly sure. It was flying at a normal plane speed you would see in the sky. And then It stoped. Looked as if it went around in a 360 in one spot. and Flew off, at an amazing speed! As it fled my sight it left a trail of light for about 5 seconds and then it was gone. ((NUFORC Note:  Witness indicates that the date of the incident is approximate.  PD));40.0217727272726;-105.258365102639
strange hovering white light that eventually descended slowly;2009-02-11 20:15:00;7  minutes;strange hovering white light that eventually descended slowly I was driving West on Highway 36 into Boulder, Colorado and I noticed a bright white light hovering above the city.  It appeared to have multiple lights clustered together as a single unit (large light on top with a couple lights on bottom).  It hovered in the same spot for about 4 minutes,  and then slowly moved down towards the foothills.  I couldnâ€™t tell whether or not the light actually went behind or in front of the flatirons before it finally went out of my sight.;40.0217727272726;-105.258365102639
A tight group of flashbulb-like lights glittering high up in a clear blue sky.;2013-01-19 14:15:00;~2 minutes;A tight group of flashbulb-like lights glittering high up in a clear blue sky We were looking at gulls riding currents very high over Walden Ponds wildlife area outside of Boulder.  They glinted white as the soared against a deep blue, cloudless sky.   Behind the birds, seemingly much higher, we noticed a peculiar group of flashing lights in a tight formation, looking much like a bunch of flashbulbs going off at the same time or like glittery fireworks.   Having forgotten binoculars, we couldn't tell if it was one very large reflective object or many smaller ones, but it appeared to remain nearly stationary for a couple of minutes.   It was fainter than the sunlit white birds but still glittery and flashbulb like against the blue.  It was small enough and faint enough in the sky that it's highly unlikely that anybody else saw it, unless, like us, their attention was attracted by the gulls.   It continued to flash constantly as it gradually became more difficult to see over a period of two or three minutes.;40.0217727272726;-105.258365102639
Loud Noise Along with UFO flashing lights aircraft.;2014-03-11 06:35:00;2 minutes;Loud Noise Along with UFO flashing lights aircraft Approximately 6:35 am Tuesday, March 11, 2014, I was in my room studying for a test while it was still dark out, with the sun just barely making the bottom of the horizon a lighter shade of blue.   Out of nowhere I heard an extremely loud humming noise coming from the sky unlike any noise I had heard before.  The noise was extremely loud and continuous and when I went outside I saw a dark aircraft with numerous blinking lights (many more than an airplane) flying over my house toward to Flatiron Mountains.   It was flying high enough that it eventually disappeared over the mountain, though it was pretty low to just go over the mountain top.  It came from the East and headed toward the West over the Flatiron Mountains.   I watched it pass over me for about a minute or so and the loud noise continued until it was no longer in sight and earshot. It was dark which made it hard to tell the shape but rather the many blinking lights on the craft were more distinguishable!   Eventually it disappeared over the mountains and the noise faded away along with the flashing lights.;40.0217727272726;-105.258365102639
Four bright symmetrical horizontal lights on a craft in sky near dusk, banked left, lights dimmed and then they vanished;2014-05-05 17:15:00;20 seconds;Four bright symmetrical horizontal lights on a craft in sky near dusk, banked left, lights dimmed and then they vanished I was jogging south when I looked up toward the southeast and toward the horizon I saw four bright lights in a symmetrical line.  Craft appeared to bank left, lights suddenly dimmed and then disappeared.  I often view the sky as an aviation enthusiast and am certain it was not a commercial craft.  The lights were very large relative to the perceived distance. It could of been a military plane of some kind but appeared to be going away from me.  Usually landing lights or other bright lights face forward.  It was there and then gone very quickly. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));40.0217727272726;-105.258365102639
8 or so red, blinking lights moving in a straight line above Boulder, CO.  ((anonymous report));2017-12-09 21:00:00;10:00;8 or so red, blinking lights moving in a straight line above Boulder, CO. 3 at first, then 8 or so red, blinking lights moving in a straight line.  Moved westward from around Denver and over the foothills. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));40.0217727272726;-105.258365102639
