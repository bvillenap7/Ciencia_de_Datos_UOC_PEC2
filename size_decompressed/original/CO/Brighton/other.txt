Craft with lights hovering over houses at night;2006-10-06 22:30:00;15 seconds;Craft with lights hovering over houses at night I was on Bridge St. in Brighton (medium sized town north of denver) when I saw an aircraft moving through the sky. It was about 10:30 at night.  I thought it was a plane, but then realized it was much too low, and also moving too slow.  It was only 10-20 feet above some houses.  It had green blinking lights on it, but I couldn't identify the shape.  There were maybe a dozen small lights, and also one larger white/yellow light. As I drove past, it lowered behind the trees and possibly landed.  It was in sight for about 15 seconds.  I wish I had gone to look closer at it but frankly I was little freaked out. This was a UFO, in that I couldn't identify what it was.  My first impression was that it was one of those flight for life helicopters.  I'm not positive that isn't what it was.  But it just seemed too big and didn't look right.  I'm sure the people in that area had to have seen it.  I am about 50% sure it was a flight-for-life, because that's my only explanation, but it just didn't seem like it.;39.948859685864;-104.816636125655
