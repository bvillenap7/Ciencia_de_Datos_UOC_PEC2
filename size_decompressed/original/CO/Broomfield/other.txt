Gold lime green irregular bulb-shaped object Broomfield, CO 5:30PM  9/27/11;2011-09-27 17:30:00;25 MINUTES;gold lime green irregular bulb-shaped object Broomfield, CO 5:30PM  9/27/11 On September 27, 2011, at about 5:30pm, I was sitting on my back patio, in Broomfield, Colorado, which is about ¼ mile east of Highway 287 and W. 10th Avenue.  Several small planes were making approaches from the north to land at Metropolitan Airport, two miles to the southwest.  While watching these planes, I noticed something about three times higher in the air.  It was the shape of a lightbulb, if the aluminum-wrapped lower base section were cut off.  It glowed gold-ish lime green.  It was moving from east to west very slowly. I went indoors and got my binoculars and watched the object through them.  A few times, it left its east-west course and veered in a northerly direction.  Then, it would go west for awhile, then, veer south before continuing westward. Initially, I thought this might be a Mylar balloon, but it did not look like one.  Also, they tend to be highly reflective.  This, instead, seemed to be glowing from within, a greenish gold color, not shiny or reflective. Then, I wondered if it were a bundle of helium balloons, but with the binoculars, I could not see any irregularity in the shape, or space between balloons where they would separate from one another.  It looked like one cohesive whole.  Its shape did not change. When it was overhead, it began moving in a more distinctly northerly direction, still going westward.  From this angle, it looked as if it had a red elongated vertical light in the very center of it, the shape of a candle wick.  This “wick” looked as if it extended about 2/3 of the height of the object.  It still glowed greenish gold around this long red central “wick.”  Also, when it got to the west of me, I thought perhaps it would not appear to be glowing (if its appearance were a matter of simply reflecting sunlight), since I would be on the side of the object that sunlight was not striking.  This was not the case, however, and the object did appear to be emitting the greenish gold color on the east side (away from the sun) as well. I watched this object for about 25 minutes.  There was quite a bit of air traffic to Metropolitan Airport (2 miles southwest) and to Denver International Airport, much further east.  The object was in fairly close proximity to two aircraft, so I am wondering if a pilot reported seeing it and could identify what it was.;39.9141244186047;-105.074105813953
Object with four flashing lights swoops over Broomfield houses.;2012-12-05 07:34:00;~1 minute;Object with four flashing lights swoops over Broomfield houses I was driving home after picking my girlfriend up for work when I noticed a large white vertical cylinder in hovering in the sky, and while I feel I could write an entire report about this incident, what followed drastically more entertaining.  As I was pulling closer to this object it started to flash, and before I could even get a camera ready to get some footage of it my girlfriend screaming out, &quotWhat the hell is that!?&quot   The object she saw was clearly a single bent saucer shape.  It had a row of four lights, with the two outside lights set slightly lower and to the side of the two in the middle.  It was incredibly low in the sky, lower than any of the planes which fly along a similar path from DIA.  The object was so low that I got a pretty good estimate of it's size, I'd say it was about 60' wide, about two houses. As soon as I saw it I turned onto a road that ran right underneath it to try to get some pictures or video of it, and as soon as I turned onto the road I saw the four lights grow brighter and the entire object just sort of swooped down over the houses and shot off towards the mountains.  I floored it to try to keep up with it, got up to over 60MPH in a residential zone trying to catch up to it and it still shot away from me like I was standing still.;39.9141244186047;-105.074105813953
Three strange lights in triangular formation, one brighter than any aircraft I have ever seen one passed overhead.;2014-01-17 18:00:00;10 minutes;Three strange lights in triangular formation, one brighter than any aircraft I have ever seen one passed overhead. I was driving home from Boulder on W 120th Avenue when I noticed 3 very peculiar lights to the southwest in a triangular formation.  One of the lights was very bright, brighter than any aircraft I have ever seen.  I continued to drive east for a few minutes when one of the objects passed several hundred feet overhead: I looked upwards and saw a craft that did not look like any aircraft I had ever seen with a very strange shape.  I had denied my having seen this craft and dismissed it as another human aircraft (an airport was a few miles to the southeast) until this evening when I discovered this website and watched Paul Hellyer's interview with Sophie Co.;39.9141244186047;-105.074105813953
Silver/metallic toroidal object seen tumbling northbound during daylight.;2020-08-08 19:50:00;15 minutes;Silver/metallic toroidal object seen tumbling northbound during daylight.   I'm an amateur astronomer and I was setting up my gear just before sundown.  When I glanced up at a plane that was leaving DIA, I noticed something else (naked eye) just north of the plane at a slightly higher altitude (~20k feet).   It looked like a shiny disk and was moving slowly due north.  I grabbed binoculars to get a closer look and I was able to see the object very clearly.  It wasn't a disk but rather toroid in shape and metallic silver.  It was tumbling as it moved north.  In diameter I would say it was approximately 100-200 feet.  The best description I can give would be a 100 foot diameter silver wedding ring being tossed in slow motion.  I grabbed my partner and gave her another pair of binos.  We just watched it for about 15 minutes until it was too far away to see with much clarity.  Because of its silver coloration I thought it might be a mylar research balloon of some kind, but I can't find any reference to such a thing online.  It had no visible lights of propulsion. ((BEGIN ADDENDUM FROM WITNESS)) Hello, I submitted a report last week regarding a sighting I experienced in Broomfield, CO on August 9th.  I posted the wrong date for the sighting as it actually occurred on Saturday August 8th.  I'd just like to make the change in case anyone else witnessed the event on the same day.   Thanks, ((END ADDENDUM));39.9141244186047;-105.074105813953
