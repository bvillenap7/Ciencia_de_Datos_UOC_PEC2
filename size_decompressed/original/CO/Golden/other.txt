Witnessed a Barbell like object that rotated irratically, changed directions several times, and finally dissapeared.;1995-06-06 18:00:00;15 minutes;Summary : Witnessed a Barbell like object that rotated irratically, changed directions several times, and finally dissapeared. I just found this web page, and when I saw the report logs, I excitedly went to June 1995. There is a report of an object seen on the 7th in another state that is remarkably similar to what I observed on June  1995. The other witnesses, who were my co-workers did not report this event, as we did not know how to go about doing so. The following is an account of what we saw. The details are not hazy in my memory, as they are burned into my mind- they could never be forgotten.I work at a Bungee jump tower in Golden CO. I am the supervisor of five employees that run the jump tower each day. At 6pm, one of my employees remarked how the clouds in the sky, when observed from the ground with the backdrop of the tower, made the tower appear to move.(the wind was blowing the clouds) I looked up, along with another employee, and noticed that this was true. Gazing at the sky behind the tower, something caught my eye. At first I thought it was an airplane, as it appeared to be at the same altitude as a plane taking off from DIA would. But then it abruptly changed directions. I pointed this out to the other employees. We then observed the object for about ten or fifteen minutes. The object was reflecting the sunlight, and therefore was flashing in the daytime sun. After five minutes or so, it appeared that the object was getting closer. We could then tell that the object appeared to be a metallic shaft with two round ends, like a barbell. The flashing was cau!sed by the object rotating irratically, and wobbling back and forth. I suggested that it was simply two mylar balloons caught in the wind, but then we noticed the same clouds previously mentioned, and the object was moving opposite these wind blown clouds, ruling out the balloon idea. Then, after approximately seven minutes of observation the object changed directions and headed back the opposite direction. The object did not seem to be getting closer at this point, but we then noticed a faint greyish foggy halo surrounding the object. At this point two customers came to the jump, and inquired as to what we were gaping at. They observed the object as well. After a maximum of fifteen tense minutes of observing this anomoly, the object changed course abruptly, went behind a small, thin, transparent cloud, and dissapeared. I would appreciate any information regarding this sitghting, i.e. whether the similar sighting in your log is of similar circumstances. Since 1995 I have been w!rithing to get this reported, and am very grateful for this service.;39.7598265306123;-105.220002040816
Low flaming meteor with possible ground impact.;2001-08-17 22:45:00;3 sec;Low flaming meteor with possible ground impact. At about 10:45 PM driving west on I-70 near the intersection of I-70 & C-470, (Golden, Colorado) I saw what was most likely a large meteor or Space Junk. It first appeared to me just below the top of my windshield at about a 30 degree angle above the horizon, appearing to flare up at that point to a bright yellow-white color with a long tail and sparks and small chunks breaking off. It was about half to 2/3rds the size of a pencil eraser at arms length. It dropped almost vertically to a point just above the ridgeline of the foothills, and appeared to dim out just before disappearing. While very hard to estimate distance, if I had to venture a guess I would say 5 to 20 miles, though it could have been much further. It looked to me that it very likely hit the ground relatively near Denver. (A call to the USGS for seismic cooberation was quite unsatisfying) I had my hand held GPS operating, and checked the current heading at that time as 201 degrees. I am trying to confirm if that is magnetic or true heading, but the GPS manual does not specify. It appears to be magnetic heading based on my aeronautical chart and the road in that area. The object was just slightly left of straight ahead, so most likely about a 195-200 degree magnetic heading from that intersection. Plotting this on my Denver Area Chart (aeronautical chart), the line runs nearly through Marshdale, Silver Springs, and Bailey Colorado. (If my GPS course turns out to be True Course, shift the line 10 Â½ degrees to the left, but magnetic course looks right on the map). This was one of the most spectacular, low and close objects of this type I have seen, and if my small plane were not in the shop I would go out looking for evidence. It may be possible if able to triangulate or get a seismic location to find a relatively pristine meteor or space debris. I don't know who to contact about meteor impacts, so perhaps you can tell me or pass this on. Perhaps NORAD/Space Command has some info on this object if they will share it with us. ((NUFORC Note:  Map sent by witness.  Curiously, no other reports from this part of Colorado for this date.  However, we presume that this is a high-quality report.  PD));39.7598265306123;-105.220002040816
Strange Noise In Golden CO Area;2002-02-19 14:10:00;15 Mins;Strange Noise In Golden CO Area Strange noise in the area of Golden.  It could have been an airraid siren, but I haven't heard of any reports that were given about an airraid siren test.  This happened around 2pm MST on 02/19/2002, we were standing outside of one of the Federal buildings when the so called 'noise' started.  It sounded like wind going through a tight crack but then it changed to sounding like a siren, we couldn't tell.  We were out in the open, and the wind never ever sounded like that.  I was looking for possible aircraft that might have flown overheard but saw nothing. Weather was partly cloudy/mid 50s and a little breezy.  Other than the wind, it was a perfect day.;39.7598265306123;-105.220002040816
Light formation in sky over Golden,CO   01/04/04;2004-01-04 20:00:00;20sec;Light formation in sky over Golden,CO   01/04/04 I was baby sitting for my brother in law, a house painter, mild mannered, father of four, on the night of Sunday, January 4 from about 8:00pm till about 11:00pm. He and his wife went to see a movie. When they returned, My brother in law Anthony offhandedly told me about some lights he saw in the sky immediatly after leaving my house for the movie. They were in a hurry because the movie was starting soon, but they stopped at the stop sign and looked up and they described seeing a hundred or so small lights kind of in an upsidedown swoosh formation. He said, and his wife Misty coroborated him, that the formation drifted slowly from easrt to west for about fifteen seconds while they stared at it. Then it slowly turned north and started morphing into more of an upsidedown U for about 3-5 more seconds before dissapearing. Shortly after the lights vanished they spotted an airplane flying eastbond for DIA traveling signifigantly faster than the light formation did. It was a dark cloudless night and we are over the ridge from Denver so we are sheltered somewhat from it's ground light. There are also no streetlights on my steet so nights are dark and starry. My personal opinion is it was something natural floating in the air, due to it's slow movement. Exhaust from an airplane, the after effects of a meteor, or a firework. ((male name deleted and Misty say it appeared to be as high or higher than the plane was. I'm just curious if any one else saw it or has any idea what it could have been.;39.7598265306123;-105.220002040816
REAL BIG BIRD OR  PTERADACTYL FLYING OVER FISHING PONDS WHILE TWO FISHERMEN WACTH IT;2004-09-30 09:30:00;night;REAL BIG BIRD OR  PTERADACTYL FLYING OVER FISHING PONDS WHILE TWO FISHERMEN WACTH IT I DONT KNOW IF ANY ONE ELSE BESIDES ME AND A BUDDY OF MINE SEEN THIS THING OR NOT BUT WHAT EVER IT IS ITS DIFFERNT THAN ANYTHING I HAVE SEEN .A BUDDY AND I WERE FISHING AT WHAT WE CALL THE TWO PONDS NEAR WHERE WE LIVE WE WERE FISHIN LATE ONE NIGHT FOR CARP AND CATFISH AND IT WAS KINDA ERIE OUT THERE WAS FOG RISEING FROM THE PONDS AND WE BOTH SEEN WHAT IT APEARED AS A PTERADACTYL FLYING OFF LIKE IF WE SCARED IT AND IT WENT OVER THE CAR DEALER SHIP WHERE WE COULD REALLY SEE IT BECUASE OF THE BRIGHT LIGHTS NO ONE BELEAVES US BUT WE DONT CARE WE KNOW WHAT WE SEEN THAT NIGHT!! IT WAS LIKE BIG REAL BIG WAY BIGGER THAN ANY OWLS ROUND HERE IT WAS LIKE A SMALL PLANE BUT I COULD NOT HEAR ANY THING WHEN IT FLEW BY US .IT HAD A LONG BODY AND LONG WINGS. ((NUFORC Note:  Date is approximate.  PD));39.7598265306123;-105.220002040816
I did not see a definite craft of sorts but heard a very distinct sound.  It felt like an engine.;2016-06-12 17:48:00;30 minutes;I did not see a definite craft of sorts, but heard a very distinct sound.  It felt like an engine.  When I say felt, I mean I was directly under the cloud cover and could tell something was above me.   Whatever it was had ample cloud cover and was moving at a steady pace with lightning and rain.  Granted this is common for the area but the sound that was traveling with the cloud was certainly an anomaly.   I certainly felt there was something above me moving just ahead of the storm. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.7598265306123;-105.220002040816
